### PR TITLE
Alcoholic report

### PR DESCRIPTION
**Pull request by adamkhalil1**

### CODE REVIEW

File: src/main/java/at/fhj/iit/cashregister/Buyable.java [hunk 1]
```diff
@@ -1,6 +1,12 @@
 package at.fhj.iit.cashregister;
 
-// TODO JavaDoc Adam
+/**
```
Comment (dev-Fabi) [inline src/main/java/at/fhj/iit/cashregister/Buyable.java:3]: Please add description

Comment (kaulex99) [inline src/main/java/at/fhj/iit/cashregister/Buyable.java:3]: General description should be added two

File: src/main/java/at/fhj/iit/cashregister/CashRegister.java [hunk 2]
```diff
@@ -8,36 +8,56 @@
...
         // TODO implement Fabi
         return 0;
     }
 
-    double getLowAlcoholicRevenue(){
-        // TODO implement Adam
-        return 0;
+    /**
+     * Total sales of low alcoholic drinks
+     *
```
Comment (kaulex99) [inline src/main/java/at/fhj/iit/cashregister/CashRegister.java:22]: Remove line break

File: src/main/java/at/fhj/iit/cashregister/CashRegister.java [hunk 3]
```diff
@@ -8,36 +8,56 @@
...
+        }
+        return lowAlcoholRevenue;
     }
 
-    double getHighAlcoholicRevenue(){
-        // TODO implement Adam
-        return 0;
+    /**
+     * Total sales of high alcoholic drinks
+     *
```
Comment (kaulex99) [inline src/main/java/at/fhj/iit/cashregister/CashRegister.java:37]: Remove line break

File: src/test/java/at/fhj/iit/cashregister/CashRegisterTest.java [hunk 4]
```diff
@@ -25,13 +31,58 @@ void getNoneAlcoholicRevenue() {
...
+        cashRegister.sell(lowAlcoholic2, "Adam", now);
+
+
+        assertEquals(0, cashRegister.getHighAlcoholicRevenue());
+
+        cashRegister.sell(highAlcoholic1, "Adam", now);
+        cashRegister.sell(highAlcoholic2, "Adam", now);
+
+        assertEquals(highAlcoholic1.getPrice() + highAlcoholic2.getPrice(), cashRegister.getHighAlcoholicRevenue());
+
```
Comment (kaulex99) [inline src/test/java/at/fhj/iit/cashregister/CashRegisterTest.java:85]: remove last line break

File: src/main/java/at/fhj/iit/cashregister/Buyable.java [hunk 5]
```diff
@@ -1,6 +1,12 @@
...
+ * @author Adam Khalil
+ * @version 1.0
+ */
+
 public interface Buyable {
     double getPrice();
+    boolean isAlcoholic();
+    double getAlcoholPercent();
```
Comment (dev-Fabi) [inline src/main/java/at/fhj/iit/cashregister/Buyable.java:11]: also add short JavaDoc to the methods

File: src/test/java/at/fhj/iit/cashregister/CashRegisterTest.java [hunk 6]
```diff
@@ -25,13 +31,58 @@ void getNoneAlcoholicRevenue() {
...
+        cashRegister.sell(nonAlcoholic2, "Adam", now);
+        cashRegister.sell(highAlcoholic1, "Adam", now);
+
+        assertEquals(0, cashRegister.getLowAlcoholicRevenue());
+
+        cashRegister.sell(lowAlcoholic1, "Adam", now);
+        cashRegister.sell(lowAlcoholic2, "Adam", now);
+
+        assertEquals(lowAlcoholic1.getPrice() + lowAlcoholic2.getPrice(), cashRegister.getLowAlcoholicRevenue());
+
```
Comment (kaulex99) [inline src/test/java/at/fhj/iit/cashregister/CashRegisterTest.java:56]: remove last linebreak

File: src/main/java/at/fhj/iit/AdamsDrink.java [hunk 7]
```diff
@@ -11,7 +11,7 @@
  */
```
Comment (kaulex99) [inline src/main/java/at/fhj/iit/AdamsDrink.java:11]: Remove line break

**Review by kaulex99 — COMMENTED** [review]
A view changes need to be done :D

File: src/test/java/at/fhj/iit/cashregister/CashRegisterTest.java [hunk 8]
```diff
@@ -25,13 +31,58 @@ void getNoneAlcoholicRevenue() {
     }
 
     @Test
```
Comment (dev-Fabi) [inline src/test/java/at/fhj/iit/cashregister/CashRegisterTest.java:33]: also add short javadoc description to test methods
```suggestion
    /**
    * Test CashRegister.getLowAlcoholicRevenue method 
    */
    @Test
```

File: src/test/java/at/fhj/iit/cashregister/CashRegisterTest.java [hunk 9]
```diff
@@ -25,13 +31,58 @@ void getNoneAlcoholicRevenue() {
...
+        assertEquals(0, cashRegister.getLowAlcoholicRevenue());
+
+        cashRegister.sell(lowAlcoholic1, "Adam", now);
+        cashRegister.sell(lowAlcoholic2, "Adam", now);
+
+        assertEquals(lowAlcoholic1.getPrice() + lowAlcoholic2.getPrice(), cashRegister.getLowAlcoholicRevenue());
+
     }
 
     @Test
```
Comment (dev-Fabi) [inline src/test/java/at/fhj/iit/cashregister/CashRegisterTest.java:59]: same here

**Review by dev-Fabi — CHANGES_REQUESTED** [review]
some minore changes for JavaDoc

**Review by dev-Fabi — APPROVED** [review]

**Pull request merged**